### PR TITLE
rename scheduler fake listers

### PR DESCRIPTION
**Pull request by zouyee**
**What type of PR is this?**
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
rename fake listers  names：

- PersistentVolumeClaimInfo to PersistentVolumeClaimLister
- CSINodeInfo to CSINodeLister
- PersistentVolumeInfo to PersistentVolumeLister
- StorageClassInfo to StorageClassLister

**Which issue(s) this PR fixes**:

Fixes #
xref: https://github.com/kubernetes/kubernetes/pull/84192

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/scheduler/algorithm/predicates/csi_volume_predicate_test.go [hunk 1]
```diff
@@ -472,8 +472,8 @@ func TestCSIVolumeCountPredicate(t *testing.T) {
 	}
 }
 
-func getFakeCSIPVInfo(volumeName string, driverNames ...string) fakelisters.PersistentVolumeInfo {
-	pvInfos := fakelisters.PersistentVolumeInfo{}
+func getFakeCSIPVLister(volumeName string, driverNames ...string) fakelisters.PersistentVolumeLister {
+	pvInfos := fakelisters.PersistentVolumeLister{}
```
Comment (Huang-Wei) [inline pkg/scheduler/algorithm/predicates/csi_volume_predicate_test.go]: pvInfos also need to be renamed.

File: pkg/scheduler/algorithm/predicates/csi_volume_predicate_test.go [hunk 2]
```diff
@@ -517,8 +517,8 @@ func getFakeCSIPVInfo(volumeName string, driverNames ...string) fakelisters.Pers
 	return pvInfos
 }
 
-func getFakeCSIPVCInfo(volumeName, scName string, driverNames ...string) fakelisters.PersistentVolumeClaimInfo {
-	pvcInfos := fakelisters.PersistentVolumeClaimInfo{}
+func getFakeCSIPVCLister(volumeName, scName string, driverNames ...string) fakelisters.PersistentVolumeClaimLister {
+	pvcInfos := fakelisters.PersistentVolumeClaimLister{}
```
Comment (Huang-Wei) [inline pkg/scheduler/algorithm/predicates/csi_volume_predicate_test.go]: same here, pvcInfos need to be renamed

File: pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go [hunk 3]
```diff
@@ -472,8 +472,8 @@ func TestCSILimits(t *testing.T) {
 	}
 }
 
-func getFakeCSIPVInfo(volumeName string, driverNames ...string) fakelisters.PersistentVolumeInfo {
-	pvInfos := fakelisters.PersistentVolumeInfo{}
+func getFakeCSIPVLister(volumeName string, driverNames ...string) fakelisters.PersistentVolumeLister {
+	pvInfos := fakelisters.PersistentVolumeLister{}
```
Comment (Huang-Wei) [inline pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go]: same here, pvInfos need to be renamed.

File: pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go [hunk 4]
```diff
@@ -517,8 +517,8 @@ func getFakeCSIPVInfo(volumeName string, driverNames ...string) fakelisters.Pers
 	return pvInfos
 }
 
-func getFakeCSIPVCInfo(volumeName, scName string, driverNames ...string) fakelisters.PersistentVolumeClaimInfo {
-	pvcInfos := fakelisters.PersistentVolumeClaimInfo{}
+func getFakeCSIPVCLister(volumeName, scName string, driverNames ...string) fakelisters.PersistentVolumeClaimLister {
+	pvcInfos := fakelisters.PersistentVolumeClaimLister{}
```
Comment (Huang-Wei) [inline pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go]: same here, pvInfos need to be renamed.

**Comment by zouyee** [comment]
@Huang-Wei  CI passed, please take a look.

**Comment by zouyee** [comment]
@ahg-g CI passed, please take a look, thanks.

**Pull request merged**